schema/encoding/jsonschema/jsonschema.go

```diff
@@ -41,7 +44,25 @@ func (ew errWriter) write(b []byte) {
 	_, ew.err = ew.w.Write(b)
 }
 
-// ValidatorToJSONSchema takes a validator and renders to JSON
+// boundariesToJSONSchema writes JSON Schema keys and values based on b, prefixed by a comma and without curly braces,
+// to w. The prefixing comma is only written if at least one key/value pair is also written.
+func boundariesToJSONSchema(w io.Writer, b *schema.Boundaries) error {
+	if b == nil {
+		return nil
+	}
+	ew := errWriter{w: w}
+
+	if !math.IsNaN(b.Min) && !math.IsInf(b.Min, -1) {
+		ew.writeFormat(`, "minimum": %s`, strconv.FormatFloat(b.Min, 'E', -1, 64))
+	}
+	if !math.IsNaN(b.Max) && !math.IsInf(b.Max, 1) {
+		ew.writeFormat(`, "maximum": %s`, strconv.FormatFloat(b.Max, 'E', -1, 64))
+	}
+	return ew.err
+}
```

Comment:
PS! This doesn't mean it can't be changed, if there is a compelling reason to change how NaN is handled by the `schema` package.